api/jobs.go

```diff
@@ -836,9 +836,10 @@ func (p *PeriodicConfig) GetLocation() (*time.Location, error) {
 
 // ParameterizedJobConfig is used to configure the parameterized job.
 type ParameterizedJobConfig struct {
-	Payload      string   `hcl:"payload,optional"`
-	MetaRequired []string `mapstructure:"meta_required" hcl:"meta_required,optional"`
-	MetaOptional []string `mapstructure:"meta_optional" hcl:"meta_optional,optional"`
+	Payload        string   `hcl:"payload,optional"`
+	MetaRequired   []string `mapstructure:"meta_required" hcl:"meta_required,optional"`
+	MetaOptional   []string `mapstructure:"meta_optional" hcl:"meta_optional,optional"`
+	TemplatePrefix string   `mapstructure:"template_prefix" hcl:"template_prefix,optional"`
```

Comment:
Super bikesheddy nitpick (😀): the field name `template_prefix` doesn't have an obvious relationship to the purpose. Maybe "id_prefix_template" or something?

Reply:
Surely this isn't the better name for this field. I'm not good with names. `id_prefix_template` sounds better, think I'll use it.